type/model/dependency.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+// SourceCD
```

Comment:
Enums can be represented as an aliased type.
Refer https://github.com/wantedly/yashima-news-timeline/blob/master/type/model/article_body_type.go